Makefile

```diff
@@ -87,7 +87,7 @@ ifneq ($(HAS_SERVER),)
 	fi; \
 
 	@echo Running golangci-lint
-	golangci-lint run ./...
+	golangci-lint run ./server/...
```

Comment:
This is because there was a node_modules package that had go files, and they were badly formatted

Reply:
We should add `./lt/...` as well then.